lib/experimental/Navigation/Tabs/index.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { action } from "@storybook/addon-actions"
+import type { Meta, StoryObj } from "@storybook/react"
+import { useState } from "react"
+import { Tabs } from "."
+
+const meta: Meta<typeof Tabs> = {
+  component: Tabs,
+  tags: ["autodocs"],
+  argTypes: {
+    type: {
```

Comment:
looks like this control does not work, I don't see it in the storybook of the PR

Reply:
I changed this now that I switched secondary to a boolean. Can you check again?